themes/default/layouts/partials/docs/search.html

```diff
@@ -1,25 +1,17 @@
 <div class="docs-type-nav-search">
     <div class="docs-type-nav">
-        {{ $mode := .Scratch.Get "mode" }}
```

Comment:
I think it's fine to remove all this check because there's no longer a "docs" (or anything other than "registry") section in this repo.

Reply:
yeah should be fine, seems to be left over from when everything was together.